.travis.yml

```diff
@@ -16,6 +16,38 @@ script:
 # Docker Run to Execute Unit E2E Selenium Package
 - echo "Running E2E Tests"
 - docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e SECRET_KEY todo-app:test Test/selenium_test.py
```

Comment:
This is module 7 but since I noticed it I feel I should say - I don't expect you need board_id, or the list ids, because the e2e test should be creating and using a test board.

Also SECRET_KEY shouldn't be needed anymore - it was for encrypting cookies (via Flask's `session` object), which I don't believe your app does anymore.